app/views/mailer/mfa_required_popular_gems_announcement.html.erb

```diff
@@ -0,0 +1,124 @@
+<% @title = @heading %>
+<% @sub_title = "👋 Hi #{@user.handle}" %>
+
+<!-- Body -->
+<table width="100%" border="0" cellspacing="0" cellpadding="0" bgcolor="#ffffff">
+  <tr>
+    <td class="content-spacing" style="font-size:0pt; line-height:0pt; text-align:left" width="20"></td>
+    <td>
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="35" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="40" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <div class="h3-1-center" style="color:#1e1e1e; font-family:Georgia, serif; min-width:auto !important; font-size:20px; line-height:26px;">
+        <p>
+          Effective today, multi-factor authentication (MFA) is required on your RubyGems account.
+        </p>
+        <br/>
+
+        <% if @user.mfa_disabled? %>
+          <p>
+            A number of operations are now restricted until you enable MFA.
+            These restrictions include editing profile pages on the web, signing in on the command line, and performing <a href="https://guides.rubygems.org/mfa-requirement-opt-in/#privileged-operations" target="_blank">privileged actions</a> (ie. push, yank, add/remove owners).
+          </p>
+          <br/>
+
+          <p> 
+            To avoid further disruptions on your account, <b>please enable multi-factor authentication 🙏.</b>
```

Comment:
```suggestion
            To re-enable these operations for your account, <b>please enable multi-factor authentication 🙏.</b>
```

It is not clear what will be further disruptions. Disabling these operations was the final thing.

Reply:
Done (https://github.com/rubygems/rubygems.org/pull/3171/commits/5f0d130a0a4a610995ef8f55fbc06a2fbe373860)